packages/discord.js/src/managers/GuildMemberRoleManager.js

```diff
@@ -174,6 +160,45 @@ class GuildMemberRoleManager extends DataManager {
     }
   }
 
+  /**
+   * Modifies the roles of the member.
+   * @param {RoleResolvable|RoleResolvable[]|Collection<Snowflake, Role>} rolesToAdd The roles to add
+   * @param {RoleResolvable|RoleResolvable[]|Collection<Snowflake, Role>} rolesToRemove The roles to remove
+   * @param {string} [reason] Reason for modifying the roles
+   * @returns {Promise<GuildMember>}
+   */
+  async modify(rolesToAdd, rolesToRemove, reason) {
```

Comment:
Wouldn't this be better if the options was an object? 
Something like: 
```suggestion
  async modify(options) {
```
```js
// 'add' the roles to add
// 'remove' the roles to remove.
// 'reason' the reason for the modification
const { add, remove, reason } = options;
```
That way you can easily include a reason without extra hassle. 🤷